chart/static-files/openapi.json

```diff
@@ -98,11 +98,11 @@
         },
         "examples": {
           "SplitsResponseNotReadyError": {
-            "summary": "The list of splits is not ready yet. Please retry later.",
+            "summary": "The server is busier than usual and the list of splits is not ready yet. Please retry later.",
             "value": "SplitsResponseNotReadyError"
           },
           "SplitsNamesError": {
-            "summary": "Cannot get the split names for the dataset.",
+            "summary": "Cannot get the split names for the dataset. Please fix your loading script.",
```

Comment:
btw, is it possible to get this error even if the dataset does not contain a loading script?